packages/native-stack/src/views/NativeStackView.native.tsx

```diff
@@ -294,6 +303,21 @@ const SceneView = ({
       onHeaderBackButtonClicked={onHeaderBackButtonClicked}
       preventNativeDismiss={isRemovePrevented} // on iOS
       onNativeDismissCancelled={onNativeDismissCancelled}
+      // @ts-expect-error this prop is available since rn-screens 3.26
+      // Unfortunately, because of the bug that exists on Fabric, where native event drivers
+      // for Animated objects are being created after the first notifications about the header height
+      // from the native side, `onHeaderHeightChange` event does not notify
+      // `animatedHeaderHeight` about initial values on appearing screens at the moment.
```

Comment:
Just one note from me, regarding the comment - I've decided not to add a TODO here, as fixing Animated.event is not related to native stack. I'll remove this comment once the Animated.event on Fabric will be fixed 👍

Reply:
There is a TODO on our end to check if it works. I believe that before releasing we'll search for TODOs so I'd suggest something like this:
`TODO: before realising a stable version, check if the following works`

Reply:
@osdnk maybe add it to our board instead 🤔